scoresly.css

```diff
@@ -0,0 +1,90 @@
+.content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  max-width: 1200px;
+}
+
+.post-credits {
+  display: flex;
+  justify-content: space-between;
+  text-wrap: wrap;
+}
+
+.left-credits, .right-credits {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  margin: 10px;
```

Comment:
```suggestion
```
Replace with `gap: 10px` on `.post-credits` and a padding, depending on what the result is you actually want. Layout should be done with a [layout-responsible parent ](https://dev.to/moruno21/dont-use-margin-in-css-33f9)as a best practice, hence using margin is quite often discouraged. It breaks component encapsulation.

In this case it doesn't really matter, but it can be good to stick to best practices.